src/web/handlers/PanGestureHandler.ts

```diff
@@ -246,9 +246,16 @@ export default class PanGestureHandler extends GestureHandler {
     }
   }
 
-  protected onPointerUp(event: AdaptedEvent): void {
+  protected onPointerUp(
+    event: AdaptedEvent,
+    ev: TouchEvent | PointerEvent
```

Comment:
In `GestureHandler` the second argument is optional, shouldn't it also be the case here?

Reply:
Marking it as optional in `GestureHandler` allowed us to call `super(event)` without passing `sourceEvent`. Here we need it to be present. However it may be confusing, (especially when calling `sourceEvent?.cancelable`), so it's good idea to also mark it as optional.

Done in [5c26a86](https://github.com/software-mansion/react-native-gesture-handler/pull/2631/commits/5c26a86fd4ccfd2a17e2d54e49d54a50c5b2843a)